packages/components/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,99 @@
+/** @jsx withSlots */
+import { compose, UseSlots, buildProps, mergeProps, withSlots } from '@fluentui-react-native/framework';
+import { requireNativeComponent } from 'react-native';
+import { IViewProps } from '@fluentui-react-native/adapters';
+import * as React from 'react';
+
+const shimmerName = 'Shimmer';
+
+const NativeShimmerView = requireNativeComponent('MSFShimmerView');
+
+/**
+ * Appearance of the views that are shimmered
+ */
+export interface IShimmerViewAppearance {
```

Comment:
This isn't obvious but fluent-web has changed its coding guidance to drop the I prefix for types, we haven't converted everything over yet but we've tried to make new code use `ShimmerViewAppearance` rather than `IShimmerViewAppearance`

Reply:
Thanks! I was confused about that as well.